specs/commandline/inspect.md

```diff
@@ -0,0 +1,271 @@
+# notation inspect
+
+## Description
+
+Use `notation inspect` command to inspect all the signatures associated with signed artifact in a human readable format.
+
+Upon successful execution, both the digest of the signed artifact and the digests of signatures manifest along with their properties associated with the signed artifact are printed in the following format:
+
+```shell
+<registry>/<repository>@<digest>
+└──application/vnd.cncf.notary.signature
+  ├──<digest_of_signature_manifest>
+    ├──<signingAlgorithm>
+    ├──<signedAttributes>
+    ├──<userDefinedAttributes>
+    ├──<unsignedAttributes>
+    ├──<certificates>
+    └──<payload>
+  ├──<digest_of_signature_manifest>
+    ├──<signingAlgorithm>
+    ├──<signedAttributes>
+    ├──<unsignedAttributes>
+    ├──<certificates>
+    └──<payload>
```

Comment:
```suggestion
<registry>/<repository>@<digest>
└── application/vnd.cncf.notary.signature
    ├── <digest_of_signature_manifest>
    │   ├── <signingAlgorithm>
    │   ├── <signedAttributes>
    │   ├── <userDefinedAttributes>
    │   ├── <unsignedAttributes>
    │   ├── <certificates>
    │   └── <payload>
    └── <digest_of_signature_manifest>
        ├── <signingAlgorithm>
        ├── <signedAttributes>
        ├── <unsignedAttributes>
        ├── <certificates>
        └── <payload>
```

Reply:
Similar format should be applied to the examples.

Reply:
Taken care